src/components/FlawForm.vue

```diff
@@ -511,12 +518,17 @@ const createdDate = computed(() => {
       />
       <div v-if="mode === 'edit'" class="border-top osim-flaw-form-section">
         <FlawComments
-          :comments="flaw.comments"
-          :taskKey="flaw.task_key"
-          :error="errors.comments"
-          :isSaving="isSaving"
+          :publicComments
+          :privateComments
+          :internalComments
+          :internalCommentsAvailable
+          :isLoadingInternalComments
+          :systemComments
+          :taskKey="flaw.task_key || ''"
+          :bugzillaLink
+          :isSaving
           @comment:addFlawComment="addFlawComment"
-          @disableForm="(value) => formDisabled = value"
+          @loadInternalComments="loadInternalComments"
```

Comment:
And this then makes sense to use the `v-bind` directive to reduce the footprint, we can also use the `v-on` directive to bind multiple events if needed
```html
<FlawComments v-bind="commentsModel" v-on="commentsEvents"  :taskKey="flaw.task_key || ''" />
```